backend/src/appointment/routes/api.py

```diff
@@ -101,6 +101,16 @@ def refresh_signature(db: Session = Depends(get_db), subscriber: Subscriber = De
         subscriber.id,
     )
 
+    # Update schedule slugs as well!
+    # This is temp until we figure this flow out
+    schedules = repo.schedule.get_by_subscriber(db, subscriber.id)
```

Comment:
Once we have multiple schedules we'll have to figure out a better flow for showing personal links and such. I assume we'll have one "General availability" schedule, but that's for future us!